sql/sql_plugin.cc

```diff
@@ -2253,6 +2253,19 @@ static bool mysql_uninstall_plugin(THD *thd, const LEX_STRING *name)
              "Stop any active semisynchronous I/O threads on this slave first.");
     goto err;
   }
+
+  /* If Group Replication is in use, the plugin can't be uninstalled.
+   * The command STOP GROUP_REPLICATION should be used before uninstall.
+  */
+  if (plugin->ref_count && !strcmp(name->str, "group_replication"))
```

Comment:
seriously

(no this is not a review comment, I just couldn't move along silently)

Reply:
https://bugs.mysql.com/bug.php?id=88597